### PR TITLE
New package: Nicotine+.Nicotine+ version 3.3.4

### PR DESCRIPTION
**Pull request by averyterrel**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163533)

### CODE REVIEW

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

File: manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- files
+- gtk
+- gtk3
+- gtk4
+- messaging
+- p2p
+- peer-to-peer
+- python
+- soulseek
+ReleaseNotesUrl: https://github.com/nicotine-plus/nicotine-plus/releases/tag/3.3.4
```
Comment (slook) [inline manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml:30]: @averyterrel For future reference the actual Release Notes file for end users is at link:

https://nicotine-plus.org/NEWS.html

File: manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+PackageVersion: 3.3.4
+PackageLocale: en-US
+Publisher: Nicotine+
+PublisherUrl: https://nicotine-plus.org/
+Author: Avery Terrel
+PackageName: Nicotine+
+PackageUrl: https://nicotine-plus.org/
+License: GPL-3.0
+LicenseUrl: https://github.com/nicotine-plus/nicotine-plus/blob/master/COPYING
+Copyright: Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```
Comment (slook) [inline manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml:14]: @averyterrel This isn't the copyright holder of this program, that is who owns the copyright the license document file.

Comment (averyterrel) [reply manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml:14]: You're right that this isn't correct, but upon further inspection, shouldn't it be `Copyright (c) 2001–2024 Nicotine+, Nicotine and PySoulSeek Contributors`? This is what is stated at the very bottom of the README, and seems accurate from what I've `grep`'d

Comment (slook) [reply manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml:14]: Yes, it looks like that would indeed be the best copyright string to use.

**Pull request merged**